finish/systemproto/pom.xml

```diff
@@ -36,6 +37,13 @@
             <scope>provided</scope>
         </dependency>
         <!-- end::grpc-stub[] -->
+        <!-- grpc compile dependencies for Jakarta EE 9+ -->
+        <dependency>
+            <groupId>org.apache.tomcat</groupId>
```

Comment:
I don't think we should ship this without Alasdair's approval.

Reply:
Perhaps liberty should bundle this annotation itself? See https://github.com/OpenLiberty/open-liberty/issues/23922

Reply:
Updated to use
```
        <dependency>
            <groupId>javax.annotation</groupId>
            <artifactId>javax.annotation-api</artifactId>
            <version>1.3.2</version>
```